src/rajAJun20/PrimeNumberOperations.java

```diff
@@ -0,0 +1,66 @@
+package rajAJun20;
+
+public class PrimeNumberOperations {
+	
+	void numberIsPrimeOrNot(int number){
+		boolean flag = false;
+		for (int index = 2; index < number/Math.sqrt(number); index++) {
+			if (number % 2 == 0) {
+				flag = true;
+				break;
+			}
+		}
+		if (flag == true) {
+			System.out.println("Number "+number+ " is not Prime");
+		}
+		else 
+			System.out.println("Number "+ number+ " is Prime");
+	}
+	
+	int totalCountOfPrimeNumbers(){
+		int count = 0;
+		for (int i = 2; i < 100; i++) {
+			boolean flag = false;
+			for (int j = 2; j < i ; j++) {
+				if(i % j == 0 ) {
+					flag = true;
+					break;
+				}
+			}
+			if(flag == false){
+				count++;
+			}
+		}
+		return count;
+	}
+
+	int sumOfPrimeNumbers(){
+		int sum = 0;
+		for (int i = 2; i < 100; i++) {
+			boolean flag = false;
+			for (int j = 2; j < i; j++) {
+				if ( i%j == 0) {
+					flag = true;
+					break;
+				}
+			}
+			if (flag == false) {
+				sum = sum + i;
+			}
+		}
+		return sum;
+	}
+
+	void avgOfPrimeNumbers(){
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
verifyNumberIsPrimeOrNot2(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count);